### ISSUE TITLE
GLKViewWidget.setCameraPosition elevation and azimuth are ignored even if rotationMethod is euler

### ISSUE THREAD

**Issue by cwiede** [pyqtgraph/pyqtgraph#1780]
### Short description
... because the function calculates a quaternion from these angles which is not used afterwards

### Additional context
To fix, change this piece of code
```
        if rotation is not None:
            # set with quaternion
            self.opts['rotation'] = rotation
        else:
            # set with elevation-azimuth, restored for compatibility
            eu = self.opts['rotation'].toEulerAngles()
            if azimuth is not None:
                eu.setZ(-azimuth-90)
            if elevation is not None:
                eu.setX(elevation-90)
            self.opts['rotation'] = QtGui.QQuaternion.fromEulerAngles(eu)
```
to this
```
        if rotation is not None:
            if elevation is not None or azimuth is not None:
                raise RuntimeError("cannot set both rotation and elevation")
            if self.opts['rotationMethod'] != "quaternion":
                eu = rotation.toEulerAngles()
                self.opts['elevation'] = eu.x() + 90
                self.opts['azimuth'] = -eu.z() - 90
            else:
                # set with quaternion
                self.opts['rotation'] = rotation
        if elevation is not None:
            self.opts['elevation'] = elevation
        if azimuth is not None:
            self.opts['azimuth'] = azimuth
        if elevation is not None or azimuth is not None and self.opts['rotationMethod'] == "quaternion":
             eu = self.opts['rotation'].toEulerAngles()
             if azimuth is not None:
                 eu.setZ(-azimuth-90)
             if elevation is not None:
                 eu.setX(elevation-90)
             self.opts['rotation'] = QtGui.QQuaternion.fromEulerAngles(eu)
```

**Comment by pijyoi** [comment]
Your fix has been incorporated into #1903.
Would you mind checking if it fixes your problem?

**Comment by cwiede** [comment]
It seems to work now (with the master branch) - thanks a lot!
